feature/selector.py

```diff
@@ -597,18 +675,18 @@ def calculate_statistics(scores: pd.DataFrame,
     scores_df = scores[columns].copy()
     selected_df = selected[columns].copy()
 
+    # Group by feature
+    scores_df = scores_df.groupby(scores_df.index).mean()
+    selected_df = selected_df.groupby(selected_df.index).mean()
+
     # Drop methods with constant scores
     if ignore_constant:
         mask = ~np.isclose(np.var(scores_df, axis=0), 0)
         scores_df = scores_df.loc[:, mask]
         selected_df = selected_df.loc[:, mask]
 
-    # Sort by index
-    scores_df.sort_index(inplace=True)
-    selected_df.sort_index(inplace=True)
-
     # Calculate statistics
-    stats_df = pd.DataFrame(index=scores.index)
+    stats_df = pd.DataFrame(index=scores_df.index)
     stats_df["_score_mean"] = scores_df.mean(axis=1)
```

Comment:
Unrelated to this PR.. there was a reason back in the time, when we had a DF with features on the column, and then we named these additional columns with "_" prefix so that we could quickly fetch/distinguish.

As of now, stats_df consists of only these columns -- shall we drop the "_" prefix ?